Content.Server/Explosion/EntitySystems/ExplosionSystem.TileFill.cs

```diff
@@ -277,7 +305,9 @@ public sealed partial class ExplosionSystem
         // diameter x diameter sized box, use a smaller box with radius sized sides:
         var box = Box2.CenteredAround(epicenter.Position, new Vector2(radius, radius));
 
-        foreach (var grid in _mapManager.FindGridsIntersecting(epicenter.MapId, box))
+        var mapGrids = new List<Entity<MapGridComponent>>();
+        _mapManager.FindGridsIntersecting(epicenter.MapId, box, ref mapGrids);
+        foreach (var grid in mapGrids)
```

Comment:
This warning is around the fact you should be storing the list on the system and clearing + re-using it instead of allocating it every time.